core/arch/arm/include/ffa.h

```diff
@@ -70,15 +74,43 @@
 #define FFA_MEM_RECLAIM			U(0x84000077)
 #define FFA_MEM_FRAG_RX			U(0x8400007A)
 #define FFA_MEM_FRAG_TX			U(0x8400007B)
+#define FFA_NOTIFICATION_BITMAP_CREATE	U(0x8400007D)
+#define FFA_NOTIFICATION_BITMAP_DESTROY	U(0x8400007E)
+#define FFA_NOTIFICATION_BIND		U(0x8400007F)
+#define FFA_NOTIFICATION_UNBIND		U(0x84000080)
+#define FFA_NOTIFICATION_SET		U(0x84000081)
+#define FFA_NOTIFICATION_GET		U(0x84000082)
+#define FFA_NOTIFICATION_INFO_GET_32	U(0x84000083)
+#define FFA_NOTIFICATION_INFO_GET_64	U(0xC4000083)
 #define FFA_SECONDARY_EP_REGISTER_64	U(0xC4000087)
 #define FFA_MEM_PERM_GET_32		U(0x84000088)
 #define FFA_MEM_PERM_GET_64		U(0xC4000088)
 #define FFA_MEM_PERM_SET_32		U(0x84000089)
 #define FFA_MEM_PERM_SET_64		U(0xC4000089)
 
+#define FFA_FEATURE_NOTIF_PEND_INTR	U(0x1)
+#define FFA_FEATURE_SCHEDULE_RECV_INTR	U(0x2)
+#define FFA_FEATURE_MANAGED_EXIT_INTR	U(0x3)
+
 /* Special value for traffic targeted to the Hypervisor or SPM */
 #define FFA_TARGET_INFO_MBZ		U(0x0)
 
+#define FFA_MSG_FLAG_FRAMEWORK		BIT(31)
+#define FFA_MSG_TYPE_MASK		GENMASK_32(7, 0)
+#define FFA_MSG_PSCI			U(0x0)
+#define FFA_MSG_SEND_VM_CREATED		U(0x4)
```

Comment:
The corresponding defines has been reviewed at https://lore.kernel.org/xen-devel/AS8PR08MB79916CDCB5DB7CA86E8077A592989@AS8PR08MB7991.eurprd08.prod.outlook.com/

It would be confusing to use different names in OP-TEE compared to Xen and I'm not so keen on making changes to patches which has already been reviewed by someone.

If you still want to rename them, would you mind proposing that in that mail thread on the xen list?

Reply:
Thanks